app/gateway/2.8.x/admin-api/developers/reference.md

```diff
@@ -0,0 +1,1417 @@
+---
+title: Developers Reference
+badge: enterprise
+---
+
+{:.note}
+> **Note:** The `/developers` API is part of the Kong Admin API, and is meant
+for bulk developer administration.
+This is not the same as the Dev Portal API [`/developer`](/gateway/{{page.kong_version}}/developer-portal/portal-api/#/operations/get-developer) endpoints,
+which return data on the logged-in developer.
+
+## Developers
+
+### List all developers
+
+**Endpoint**
+
+<div class="endpoint get">/developers</div>
+
+**Response**
+
+```
+HTTP/1.1 200 OK
+```
+
+```json
+{
+    "data": [
+        {
+            "consumer": {
+                "id": "3f0ad41f-39b5-4fd0-ad1e-0b779fcee35c"
+            },
+            "created_at": 1644615612,
+            "email": "some-email@example.com",
+            "id": "3c6d4d8c-db86-4236-800b-8911e788425b",
+            "meta": "{\"full_name\":\"Barry\"}",
+            "roles": [
+                "application developer"
+            ],
+            "status": 0,
+            "updated_at": 1644615612
+        },
+        {
+            "consumer": {
+                "id": "9dbd764c-9548-4c9f-a4d1-6ea11e32a803"
+            },
+            "created_at": 1644615583,
+            "email": "some-other-email@example.com",
+            "id": "5f60930a-ad12-4303-ac5a-59d121ad4942",
+            "meta": "{\"full_name\":\"Diana\"}",
+            "roles": [
+                "application developer"
+            ],
+            "status": 0,
+            "updated_at": 1644615583
+        }
+    ],
+    "next": null,
+    "total": 2
+}
+```
+
+### Create a developer account
+
+**Endpoint**
+
+<div class="endpoint post">/developers</div>
+
+**Request body**
+
+Attribute                     | Description
+---------:                    | --------   
+`meta`                        | Metadata for the account in JSON format. Accepts fields defined in the Dev Portal settings. <br><br> By default, the meta attribute requires a `full_name` field. You can remove this requirement, or add other fields as neccessary. <br><br> For example: `meta: {"full_name":"<NAME>"}`.
+`email` <br>*required*        | The email of the developer to create. This becomes their login username.
+`password`<br>*semi-optional* | Create a password for the developer. Required if basic authentication is enabled.
+`key` <br>*semi-optional*     | Assign an API key to the developer. Required is key authentication is enabled.
+`id`                          | A consumer UUID to associate this developer account with. [TO DO: figure out what this actually does.]
```

Comment:
Is this an attribute that users can actually/should set, or should it left out?

Reply:
This is actually the `developer` entity ID, not the consumer ID. The consumer is created automatically when creating a new developer and there isn't a way to link the new developer to an existing consumer. You can, however, specify your own UUID with this `id` property if you want.